source/api/commands/type.md

```diff
@@ -121,7 +120,7 @@ Using `.type()` on a date input (`<input type="date">`) requires specifying a va
 
 This isn't exactly how a user would type into a date input, but is a workaround since date input support varies between browsers and the format varies based on locale. `yyyy-MM-dd` is the format required by {% url "the W3 spec" https://www.w3.org/TR/html/infrastructure.html#sec-dates %} and is what the input's `value` will be set to regardless of browser or locale.
 
-Special characters (`{leftarrow}`, `{selectall}`, etc) are not permitted.
+Special characters (`{leftarrow}`, `{selectall}`, etc.) are not permitted.
```

Comment:
`Minor` `Style`

`etc.` should always end with a period unless it'd double up the periods; forex, at the end of a sentence.